addons/website/static/src/js/editor/editor.js

```diff
@@ -5,6 +5,14 @@ var weWidgets = require('web_editor.widget');
 var wUtils = require('website.utils');
 
 weWidgets.LinkDialog.include({
+    events: _.extend({}, weWidgets.LinkDialog.prototype.events || {}, {
+        'change select[name="link_anchor"]': '_onAnchorChange',
+        'input input[name="url"]': '_onURLInput',
+    }),
+    custom_events: _.extend({}, weWidgets.LinkDialog.prototype.events || {}, {
```

Comment:
You did not adapt here :/ You write `events` inside `custom_events`...

Reply:
Sorry focused on adding `|| {}`